app/src/main/java/pl/edu/zut/mad/appwizut2/fragments/CaldroidCustomFragment.java

```diff
@@ -198,7 +198,7 @@ public void onDataLoaded(List<ListItemContainer> data) {
                 }
             }
             HashMap<String, Object> extraData = getExtraData();
-            extraData.put("EVENTS", mEventCountsOnDays);
+            extraData.put(Constants.EVENTS, mEventCountsOnDays);
```

Comment:
If this is becoming constant then:
- Extract it also in place where data is retrieved from map (in CaldroidCustomAdapter)
- Give it better name and doc comment (this constant doesn't hold events)